pkg/selinux/setroubleshoot-app.js

```diff
@@ -0,0 +1,7 @@
+require([
+            "selinux/setroubleshoot",
+        ],
+function(troubleshoot) {
+    const app = document.getElementById('app');
+    troubleshoot.init(app);
+});
```

Comment:
Seems like this file should be merged with setroubleshoot.js ... that's what I've been doing with all the other places where we're moving code out of the HTML due to CSP.